tests/unit/test_utils.py

```diff
@@ -99,3 +98,24 @@ def test_flatten_list(value: List, anticpated_result: List) -> None:
     """test for flatten list"""
     actual_result = utils.flatten_list(value)
     assert list(actual_result) == anticpated_result
+
+
+@pytest.mark.parametrize(
+    "value, anticpated_result",
+    [(1, "1s"), (60 + 1, "1m1s"), (3600 + 60 + 1, "1h1m1s"), (86400 + 3600 + 60 + 1, "1d1h1m1s")],
+    ids=["seconds", "minutes seconds", "hours minutes seconds", "days hours minutes seconds"],
+)
+def test_human_time(value: Union[int, float], anticpated_result: str) -> None:
+    """test for human time"""
```

Comment:
What does it text?

Reply:
it a unit test for the function called human time in the util file, that func:

```
"""convert seconds into human readable
    00d00h00m00s format"""
```
I'll add this specifically